docs/DeveloperGuide.md

```diff
@@ -161,20 +162,34 @@ How the parsing works:
 
 The `Model` component,
 
-* stores the address book data i.e., all `Customer` objects (which are contained in a `UniqueCustomerList` object). The
-  address book is exposed to the outside as a `ReadOnlyBook` objects.
-* stores the delivery book data i.e., all `Delivery` objects (which are contained in a `UniqueDeliveryList` object). The
-  delivery book is exposed to the outside as a `ReadOnlyBook` objects.
+* stores the address book data i.e., all `Customer` objects. (See the [ReadOnlyBook Model](#ReadOnlyBook-Model) section
+  below for
+  more details)
+* stores the delivery book data i.e., all `Delivery` objects. (See the [ReadOnlyBook Model](#ReadOnlyBook-Model) section
+  below for
+  more details)
 * stores the currently filtered `Customer` objects (e.g., results of a search query) as a separate _filteredCustomers_
-  list
+  list.
 * stores the currently filtered `Delivery` objects (e.g., results of a status filter query) as a separate
-  _filteredDeliveries_ list
+  _filteredDeliveries_ list.
+* stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as
+  a `ReadOnlyUserPref` objects.
 * stores the currently sorted `Delivery` objects (e.g., results of a sort query) as a separate _sortedDeliveries_
   list
 * stores an unmodifiable `ObservableList<ListItem>` that can be 'observed' e.g. the UI can be bound
   to this list so that the UI automatically updates when the data in the list change.
```

Comment:
Perhaps can mention about the `user` object as well as it appears in the class diagram